testing/communication/EndpointShape.ttl

```diff
@@ -184,3 +184,4 @@ shapes:GenericEndpointShape
 			sh:severity sh:Violation ;
 			sh:message "<https://raw.githubusercontent.com/International-Data-Spaces-Association/InformationModel/master/testing/communication/EndpointShape.ttl> (GenericEndpointShape): An ids:genericEndpointConfiguration property must point from an ids:GenericEndpoint to an xsd:string."@en ;
 		] ;
+		.
```

Comment:
I missed the end point in the endpoint.. shame on me..

Thanks!

Reply:
just leave a notice for myself: the content of the message here (maxcount:1) needs to be corrected.